application/src/main/java/org/togetherjava/tjbot/commands/help/BookmarksSystem.java

```diff
@@ -0,0 +1,185 @@
+package org.togetherjava.tjbot.commands.help;
+
+import net.dv8tion.jda.api.EmbedBuilder;
+import net.dv8tion.jda.api.entities.MessageEmbed;
+import net.dv8tion.jda.api.entities.channel.ChannelType;
+import net.dv8tion.jda.api.entities.channel.concrete.ThreadChannel;
+import net.dv8tion.jda.api.entities.channel.unions.MessageChannelUnion;
+import net.dv8tion.jda.api.events.interaction.command.GenericCommandInteractionEvent;
+
+import org.togetherjava.tjbot.config.Config;
+import org.togetherjava.tjbot.db.Database;
+import org.togetherjava.tjbot.db.generated.tables.records.BookmarksRecord;
+
+import javax.annotation.Nullable;
+
+import java.awt.*;
+import java.time.Duration;
+import java.time.Instant;
+import java.util.List;
+import java.util.Objects;
+import java.util.function.Consumer;
+import java.util.function.Predicate;
+import java.util.regex.Pattern;
+
+import static org.togetherjava.tjbot.db.generated.tables.Bookmarks.BOOKMARKS;
+
+/**
+ * The bookmarks system provides methods to interact with the database. It also enables the
+ * {@link BookmarksCommand} to request paginations from the {@link BookmarksPaginatorInteractor}.
+ */
+public final class BookmarksSystem {
+
+    public static final Color COLOR_SUCCESS = new Color(166, 218, 149);
+    public static final Color COLOR_WARNING = new Color(245, 169, 127);
+    public static final Color COLOR_FAILURE = new Color(238, 153, 160);
+
+    public static final int MAX_BOOKMARK_COUNT_TOTAL = 1_000_000;
+    public static final int WARN_BOOKMARK_COUNT_TOTAL = 900_000;
+    public static final int MAX_BOOKMARK_COUNT_USER = 500;
```

Comment:
this constant is a bit ambigious. sounds as if 500 is the max amount of users supported. but its the bookmark limit **per** user. maybe `MAX_BOOKMARK_COUNT_PER_USER`